Resources/Prototypes/_LostParadise/Entities/Objects/Misc/Papers/folder.yml

```
@@ -122,13 +122,13 @@
 
 - type: entity
   id: LPPBoxFolderUSSP
-  name: Папка СССП
+  name: USSP Folder
   parent: BoxFolderBase
-  description: Жалкая куча реформ СССП!
+  description: A pitiful pile of USSP reforms!
   components:
   - type: Sprite
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)